targets/TARGET_STM/mbed_rtx.h

```diff
@@ -137,4 +137,13 @@
 #define MBED_CONF_RTOS_MAIN_THREAD_STACK_SIZE 3072
 #endif
 
+#if (defined(TARGET_STM32L475VG) || defined(TARGET_STM32L443RC))
+#if defined(__ARMCC_VERSION)
+    extern uint32_t               Image$$ARM_LIB_HEAP$$ZI$$Base[];
+    extern uint32_t               Image$$ARM_LIB_HEAP$$ZI$$Length[];
+    #define HEAP_START            Image$$ARM_LIB_HEAP$$ZI$$Base
```

Comment:
There used to have this kind of of definition depending on toolchain.
But it has been removed some time ago.
So I don't think adding this back depending on toolchain and targets is a so good thing?

Reply:
This definition is needed to over write the defaults used in retarget and toolchain specific file. Earlier the changes were present for 2-region memory model for specific target,  with change in memory model of Mbed os this was not needed and hence removed. This addition also indicates that the linker script here does not follow the default memory model, which is stack at the bottom of RAM memory